web/client/codechecker_client/cmd_line_client.py

```diff
@@ -839,10 +839,9 @@ def handle_diff_results(args):
 
     init_logger(args.verbose if 'verbose' in args else None, stream)
 
-    require_export_dir = any([o in ['html', 'gerrit', 'codeclimate']
-                              for o in args.output_format])
-    if require_export_dir and 'export_dir' not in args:
-        LOG.error("No export directory given!")
+    if len(args.output_format) > 1 and ('export_dir' not in args):
+        LOG.error("Export directory is required if multiple output formats "
```

Comment:
I think we should extend the help of this command too with this information. And mention that if only 1 output is given, the results will be printed to the standard output.

Reply:
done